cmd/soroban-cli/src/commands/contract/arg_parsing.rs

```diff
@@ -63,12 +68,20 @@ pub fn build_host_function_parameters(
     cmd.build();
     let long_help = cmd.render_long_help();
 
-    // get_matches_from exits the process if `help`, `--help` or `-h`are passed in the slop
+    // try_get_matches_from returns an error if `help`, `--help` or `-h`are passed in the slop
     // see clap documentation for more info: https://github.com/clap-rs/clap/blob/v4.1.8/src/builder/command.rs#L631
```

Comment:
```suggestion
    // see clap documentation for more info: https://github.com/clap-rs/clap/blob/v4.1.8/src/builder/command.rs#L586
```

Reply:
this is just a change to the line number where it mentions that `try_get_matches_from` doesn't exit the process, but returns an error